weave

```diff
@@ -1863,6 +1871,8 @@ deprecation_warnings() {
 COMMAND=$1
 shift 1
 
+[ "$COMMAND" = "run" ] || handle_help_arg "$@"
```

Comment:
just a general principle that we allow arguments to come in any order.

(although this isn't true, for instance, for `--without-dns`)